src/components/mission.module.css

```diff
@@ -0,0 +1,7 @@
+.tableContainer {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly add space between the nav and the missions table.
- [ ] [OPTIONAL] Please make the table width 100% irrespective of the content in the table to look like the template.